### DISCUSSION TITLE
Real time db

### DISCUSSION THREAD

**Discussion by sdekna** [Budibase/budibase#6591] [Help]
Can I use the platform to create a map tracker that tracks locations of users in real time... location will be updated constantly from user phone, and then can be distribute it throughout the apps?

**Comment by melohagan** [comment]
HI @sdekna 

Not at the moment. There is no functionality in Budibase for handling sub/pub, websockets, or other real time 'live' data functionality.

**Comment by tmdoit** [comment]
You can consider https://rxdb.info, could be nice to have real-time UI :)
